hs-abci-sdk/src/Tendermint/SDK/Types/Transaction.hs

```diff
@@ -38,12 +39,14 @@ data RawTransaction = RawTransaction
   , rawTransactionSignature :: ByteString
   } deriving Generic
 
+-- basically also a message type
 instance Serialize.Serialize RawTransaction
 
 parseRawTransaction
```

Comment:
This use of Data.Serialize is a fine stopgap, but go ahead and make an issue to write a protobuf file for this in the sdk itself so we can get away from it.

Reply:
See #117